<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QMainWindow" name="window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Control</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/icon_remote_control.png</normaloff>Images/icon_remote_control.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane {
    border-top: 0px;
}
     
QTabBar::tab {
    border: 1px solid #333;
    border-radius: 2px;
    color: #333;
    padding: 2px 2px 0px 5px;
    margin: 0px 0px 0px 0px;
    background: #fff;
}
     
QTabBar::tab:selected {
    border-color: #000;
    background: #f1f1f1;
}</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Icons Mode</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Icons</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>1184</width>
                <height>561</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="2" column="0">
                <widget class="QPushButton" name="icons4">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>Images/Icons/icon1.png</normaloff>Images/Icons/icon1.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>72</width>
                   <height>72</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="icons5">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>Images/Icons/icon9.png</normaloff>Images/Icons/icon9.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>72</width>
                   <height>72</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QPushButton" name="icons6">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>Images/Icons/icon8.png</normaloff>Images/Icons/icon8.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>72</width>
                   <height>72</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QPushButton" name="icons8">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>Images/Icons/icon7.png</normaloff>Images/Icons/icon7.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>72</width>
                   <height>72</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QPushButton" name="icons7">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>Images/Icons/icon2.png</normaloff>Images/Icons/icon2.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>72</width>
                   <height>72</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="icons1">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>Images/Icons/icon5.png</normaloff>Images/Icons/icon5.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>72</width>
                   <height>72</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="icons2">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>Images/Icons/icon6.png</normaloff>Images/Icons/icon6.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>72</width>
                   <height>72</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QPushButton" name="icons3">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>Images/Icons/icon4.png</normaloff>Images/Icons/icon4.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>72</width>
                   <height>72</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="icons0">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>Images/Icons/icon3.png</normaloff>Images/Icons/icon3.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>72</width>
                   <height>72</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Fonction</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QPushButton" name="fonction1">
             <property name="text">
              <string>Bouger les
doigts selon
 le temps</string>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>fonction1</zorder>
          <zorder>groupBox_6</zorder>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Slider Mode</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Pouce</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSlider" name="qslider4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Index</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QSlider" name="qslider3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Majeur</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QSlider" name="qslider2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Annulaire</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QSlider" name="qslider1">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QGroupBox" name="groupBox">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="title">
           <string>Auriculaire</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QSlider" name="qslider0">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1248</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuiter"/>
   </widget>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionConfigurer_la_connexion"/>
    <addaction name="actionConfigurer_les_raccourcis_clavier"/>
    <addaction name="separator"/>
    <addaction name="actionConnexion"/>
    <addaction name="actionDeconnexion"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionManuel_d_utilisation_de_RC"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuConfiguration"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuiter">
   <property name="icon">
    <iconset>
     <normalon>Images/icone_quitter.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Quiter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionManuel_d_utilisation_de_RC">
   <property name="icon">
    <iconset>
     <normaloff>Images/help-23413f.png</normaloff>Images/help-23413f.png</iconset>
   </property>
   <property name="text">
    <string>Manuel d'utilisation de RC</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionConfigurer_la_connexion">
   <property name="icon">
    <iconset>
     <normalon>Images/configurer-icone-7451-96.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Configurer la connexion</string>
   </property>
  </action>
  <action name="actionConfigurer_les_raccourcis_clavier">
   <property name="icon">
    <iconset>
     <normalon>Images/configurer-icone-7451-96.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Configurer les raccourcis clavier</string>
   </property>
  </action>
  <action name="actionConnexion">
   <property name="text">
    <string>Connexion</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDeconnexion">
   <property name="text">
    <string>Déconnexion</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
